test/data/validation/destinations/criteo_audience.json

```diff
@@ -1,34 +1,34 @@
 [
   {
     "config": {
-      "audienceId": {
-        "cloud": "000000"
-      },
+      "audienceId": "123",
```

Comment:
Was this field source specific earlier or just a miatake in the tests?

Reply:
@yashasvibajpai I have made the changes as we discussed. Can you review it again?